### ISSUE TITLE
Incorrect documentation example for Gdax.WebsocketClient (incorrect endpoint)

### ISSUE THREAD

**Issue by dereksan** [coinbase/gdax-node#264]
[README.md WebSocket Client](https://github.com/coinbase/gdax-node/blob/master/README.md#websocket-client) has incorrect example websocket endpoint for sandbox environment. The endpoint should be `wss://ws-feed-public.sandbox.gdax.com` but it has the rest sandbox endpoint `https://api-public.sandbox.gdax.com`

The example has:
```
    const websocket = new Gdax.WebsocketClient(
      ['BTC-USD', 'ETH-USD'],
      'https://api-public.sandbox.gdax.com',
      {
        key: 'suchkey',
        secret: 'suchsecret',
        passphrase: 'muchpassphrase',
      },
      { channels: ['full', 'level2'] }
      );
```

But should be:
```
    const websocket = new Gdax.WebsocketClient(
      ['BTC-USD', 'ETH-USD'],
      'wss://ws-feed-public.sandbox.gdax.com',
      {
        key: 'suchkey',
        secret: 'suchsecret',
        passphrase: 'muchpassphrase',
      },
      { channels: ['full', 'level2'] }
    );
```

**Comment by rmm5t** [comment]
Nice catch. Please submit a PR to fix this.

**Comment by dereksan** [comment]
Created pull request: https://github.com/coinbase/gdax-node/pull/266

**Comment by fb55** [comment]
Fixed in #266
